protocols/swap/contracts/Swap.sol

```diff
@@ -123,7 +119,7 @@ contract Swap is ISwap {
         * thus determines whether the msg.sender is an authorized sender.
         */
       if (msg.sender != _order.sender.wallet) {
-        require(isAuthorized(_order.sender.wallet, msg.sender),
+        require(isSenderAuthorized(_order.sender.wallet, msg.sender),
```

Comment:
could make this consistent with the require below. i.e. remove the `if` statement on 121, and add that case into `isSenderAuthorized`. Just like it is for `isSignerAuthorized`